rootfs/api/tests/test_auth.py

```diff
@@ -248,7 +248,7 @@ def test_passwd(self):
         }
         response = self.client.post(url, submit,
                                     HTTP_AUTHORIZATION='token {}'.format(token))
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 401)
```

Comment:
Because it is a failed authentication and not a bad request - we were using the wrong status code